whoami/whoami.go

```diff
@@ -61,6 +61,17 @@ var (
 	//fatal = log.New(os.Stderr, "", log.Lshortfile)
 )
 
+func lookupUserName() string {
```

Comment:
If it's a small command that does one thing, it's not a bad idea to do something like this: https://github.com/EricLagerg/go-coreutils/commit/72059051dd6634d912ddfbebb9d35e7f5e659e31